FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -160,19 +159,7 @@ extension EncryptedStorage {
 
 // MARK: - Keys
 extension EncryptedStorage {
-    func addKeys(keyDetails: [KeyDetails], passPhrase: String?, source: KeySource, for email: String) {
-        guard let user = getUserObject(for: email) else {
-            logger.logError("Can't find user with given email to add keys. User should be already saved")
-            return
-        }
-        try! storage.write {
-            for key in keyDetails {
-                storage.add(try! KeyInfoRealmObject(key, passphrase: passPhrase, source: source, user: user))
-            }
-        }
-    }
-
-    func updateKeys(keyDetails: [KeyDetails], passPhrase: String?, source: KeySource, for email: String) {
+    func putKeypairs(keyDetails: [KeyDetails], passPhrase: String?, source: KeySource, for email: String) {
         guard let user = getUserObject(for: email) else {
```

Comment:
I've noticed yet another issue, but that is more complicated and should be in another issue (requires migration). The private keys should have a unique index on fingerprint+email, not just fingerprint. Because if it's just fingerprint, I can't use the same key for two accounts.

Reply:
Realm has following limitation:
> You can define only one primary key per object schema.

So we need to think how to get that index with fingerprint and email.

Reply:
probably a primary id that has both email and fingerprint in the string.